package.json

```diff
@@ -33,6 +33,7 @@
   },
   "devDependencies": {
     "concurrently": "^3.6.1",
+    "decache": "^4.5.1",
```

Comment:
What do you think about using `import-fresh` (https://www.npmjs.com/package/import-fresh) here instead? The adoption on it seems broader and would be a single call instead of two as with `decache`.

Reply:
This seems to cause issues elsewhere that decache doesn't.  I tried it out this morning.  I'm able to install the import-fresh module and the test pass.  The moment i switch a require call to an importFresh call, I get the error: 

`  1) Authentication
       when logged in
         "before all" hook:
     TypeError: Cannot stub non-existent own property session
      at Sandbox.stub (node_modules/sinon/lib/sinon/sandbox.js:288:19)
      at Context.before (test/functional/userAuth.test.js:44:24)`